.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
```

Comment:
That's not a bad idea. There's a bunch of other JS in the repo that won't comply to this, but we don't need to have CI fail on eslint failures right away..